testsuite/features/step_definitions/lock_packages_on_client.rb

```diff
@@ -1,11 +1,12 @@
 # Copyright (c) 2010-2019 SUSE LLC.
 # Licensed under the terms of the MIT license.
 
-Then(/^"(.*?)" is locked on this client$/) do |pkg|
+Then(/^"(.*?)" is locked on "(.*?)"$/) do |pkg, system|
+  node = get_target(system)
   zypp_lock_file = '/etc/zypp/locks'
   raise "File already exist: #{zypp_lock_file}" unless file_exists?($client, zypp_lock_file)
```

Comment:
Should this be an `if` ?

Reply:
It seems this comes from a time where we were self parsing "/etc/zypp/locks". There it was required that the file exists.
But now we use a zypper command and this works perfectly fine without the file.
I remove this test